modules/mobile-agents/android/jax-rs/src/main/webapp/WEB-INF/cxf-servlet.xml

```diff
@@ -23,7 +23,7 @@
          http://www.springframework.org/schema/beans http://www.springframework.org/schema/beans/spring-beans-2.0.xsd
          http://cxf.apache.org/jaxrs http://cxf.apache.org/schemas/jaxrs.xsd">
 
-    <jaxrs:server id="operationService" address="/operations">
+    <jaxrs:server id="operationService" address="/operation">
```

Comment:
/operations can be a better name than /operation

For ex: When getting pending-operations list, /operations/{device-id} sounds much better than /operation/{device-id}